pkgs/tools/misc/moar/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, buildGoModule, fetchFromGitHub, installShellFiles }:
+
+buildGoModule rec {
+  pname = "moar";
+  version = "1.10.0";
+
+  src = fetchFromGitHub {
+    owner = "walles";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-cFXUspVSCUy0q5CW8K+YL/LBpK87qlPys8hg6AYvg5M=";
+  };
+
+  vendorSha256 = "sha256-RfkY66879Us0UudplMzW8xEC1zs+2OXwyB+nBim3I0I=";
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  postInstall = ''
+    installManPage ./moar.1
+  '';
+
+  meta = with lib; {
+    description = "Nice-to-use pager for humans";
+    homepage = "https://github.com/walles/moar";
+    license = licenses.bsd2; # More specifically, BSD2 with views sentence.
```

Comment:
Could you add the license similar to [this](https://github.com/NixOS/nixpkgs/blob/b2444828a1a6682a3976d8ab9758b26bca9a7592/lib/licenses.nix#L152-L155)?

Reply:
Sure thing! Does it have to be in a different PR or this PR is enough?

Reply:
Just add it in this one; as a separate commit of course.